packages/cf-typography/usage.md

```diff
@@ -23,8 +23,8 @@ and has more basic typography patterns.
     - [Meta header](#meta-header)
 - [Link patterns](#link-patterns)
     - [Links with icons](#links-with-icons)
-    - [Jump link](#jump-link)
     - [Block link](#block-link)
+    - [Jump link](#jump-link)
```

Comment:
Order is wrong here. Jump links appear first.

Reply:
Fixed in amended commit by eliminating the block link entry from the TOC entirely. Since it's not something that should be used directly, one shouldn't need to be looking for it in the TOC.